src/briefcase/commands/base.py

```diff
@@ -887,16 +882,57 @@ def update_cookiecutter_cache(self, template: str, branch="master"):
         if is_repo_url(template):
             # The app template is a repository URL.
             #
-            # When in `no_input=True` mode, cookiecutter deletes and reclones
-            # a template directory, rather than updating the existing repo.
+            # When in `no_input=True` mode, cookiecutter deletes and reclones a template
+            # directory, rather than updating the existing repo.
```

Comment:
I have a *very* vague recollection that it might have been a *long* time ago, when we were doing a lot more reliance on Cookiecutter to manage repo clones... but it definitely isn't true now. I'll remove the comment.